src/components/Sidebar/Sidebar.js

```diff
@@ -1,46 +1,87 @@
-import React from 'react'
+import React, { useCallback, useMemo } from 'react'
+import { useHistory } from 'react-router-dom'
 import { GU, useTheme, Link } from '@1hive/1hive-ui'
 import gardensLogo from '@assets/gardensLogoMark.svg'
+import defaultGardenLogo from '@assets/defaultGardenLogo.png'
 import MenuItem from './MenuItem'
+import { useGardens } from '@/providers/Gardens'
 
-function Sidebar({ gardens, activeGarden, onOpen }) {
+function Sidebar() {
   const theme = useTheme()
+  const history = useHistory()
+  const { connectedGarden, gardens } = useGardens()
+
+  const handleSelectGarden = useCallback(
+    path => {
+      history.push(path)
+    },
+    [history]
+  )
```

Comment:
This can be removed right? Since we are using the `href` with the constructed path in the MenuItem component. Also we can remove the `onOpen` prop